examples/resolvers-fauna/README.md

```diff
@@ -0,0 +1,12 @@
+# Grafbase ⨯ Fauna
+
+This examples hows how to create a GraphQL API using Grafbase Resolvers that can read and write data to a MySQL database hosted by Fauna &mdash; [Read the guide](https://grafbase.com/guides/build-and-deploy-a-graphql-api-to-the-edge-with-fauna)
```

Comment:
```suggestion
This examples shows how to create a GraphQL API using Grafbase that can read and write data to a MySQL database hosted by Fauna &mdash; [Read the guide](https://grafbase.com/guides/build-and-deploy-a-graphql-api-to-the-edge-with-fauna)
```

Reply:
Good catch! I just spotted this inside PlanetScale README.